src/icon-manifest/supportedExtensions.ts

```diff
@@ -476,6 +476,7 @@ export const extensions: IFileCollection = {
       filename: true,
       format: FileFormat.svg,
     },
+    { icon: 'stylable', extensions: ['.st.css'], languages: [languages.stylable], light: true, format: FileFormat.svg },
```

Comment:
Please remove `light` attribute. This is used if there are different icons for dark and light themes.

Reply:
👍